pages/country/_country.vue

```diff
@@ -91,6 +111,14 @@ export default {
     }
   },
 
+  computed: {
+    countryName() {
+      const countryToFind = this.$route.params.country
+      const countryEntry = COUNTRIES.find(country => country.urlAliases.includes(countryToFind));
+      return countryEntry?.name
```

Comment:
@afifsohaili Hi I think the v3 API is expecting country code instead of country name for query parameter. may need to update the code in api too.